### ISSUE TITLE
Error when running npm-groovy-lint Docker image: Unable to use CodeNarc JSON result

### ISSUE THREAD

**Issue by CliveUnger** [nvuillam/npm-groovy-lint#369]
I am trying to use the npm-groovy-lint docker image in a GitLab CI pipeline. The `.gitlab-ci.yml` configuration looks something like the below (similar structure to the Circle CI example given):
```
groovy_lint:
  stage: lint
  image: 
    name: nvuillam/npm-groovy-lint
    entrypoint: 
      - /bin/bash
  script:
    - pwd; ls;
    - node -v
    - java --version
    - npm-groovy-lint --version
    - npm-groovy-lint --noserver --config ".groovylintrc.json" --failon warning --verbose
```

For some reason the CodeNarc server does not start and I get the following error:

```
$ node -v
v18.19.1
$ java --version
openjdk 17.0.10 2024-01-16
OpenJDK Runtime Environment (build 17.0.10+7-alpine-r0)
OpenJDK [64](https://gitlab-master.nvidia.com/dl/compiler_verification/myelin-bloom-jenkins-shared-library/-/jobs/89272342#L64)-Bit Server VM (build 17.0.10+7-alpine-r0, mixed mode, sharing)
$ npm-groovy-lint --version
GroovyLint: Unable to start CodeNarc Server. Use --noserver if you do not even want to try
GroovyLint: Successfully processed CodeNarc: 
npm-groovy-lint version 14.4.0
Embeds:
Error collecting CodeNarc version
- Groovy version 3.0.9 (superlite)
$ npm-groovy-lint --noserver --config ".groovylintrc.json" --failon warning --verbose
Unable to use CodeNarc JSON result
undefined
Unexpected error: Cannot convert undefined or null to object
TypeError: Cannot convert undefined or null to object
    at Function.keys (<anonymous>)
    at processOutput (/usr/local/lib/node_modules/npm-groovy-lint/lib/output.js:109:37)
    at NpmGroovyLint.postProcess (/usr/local/lib/node_modules/npm-groovy-lint/lib/groovy-lint.js:335:39)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async NpmGroovyLint.run (/usr/local/lib/node_modules/npm-groovy-lint/lib/groovy-lint.js:[69](https://gitlab-master.nvidia.com/dl/compiler_verification/myelin-bloom-jenkins-shared-library/-/jobs/89272342#L69):13)
    at async /usr/local/lib/node_modules/npm-groovy-lint/lib/index.js:12:9
```

I tried some different permutations of this config, different file paths, no flags, tried older versions of the image even, but always the same error. Some guidance on this error would be greatly appreciated!
